### PR TITLE
Refactor and clean up e2e framework utils, this patch handles test/e2e/framework/kubelet_stats.go file

### PR DESCRIPTION
**Pull request by WanLinghao**
/kind cleanup

What this PR does / why we need it:
Refactor and clean up e2e framework utils, this patch handles test/e2e/framework/kubelet_stats.go file
ref: https://github.com/kubernetes/kubernetes/issues/76206
```release-note
NONE
```

### CODE REVIEW

**Comment by WanLinghao** [comment]
rebase after https://github.com/kubernetes/kubernetes/pull/80708 merged

File: hack/.golint_failures [hunk 1]
```diff
@@ -559,6 +559,7 @@ staging/src/k8s.io/sample-apiserver/pkg/apis/wardle/v1alpha1
 staging/src/k8s.io/sample-apiserver/pkg/registry/wardle/fischer
 staging/src/k8s.io/sample-apiserver/pkg/registry/wardle/flunder
 test/e2e/common
+test/e2e/framework/kubelet
```
Comment (cblecker) [inline hack/.golint_failures]: Why are we introducing lint failures in a refactor?

Comment (WanLinghao) [reply hack/.golint_failures]: @cblecker Updated, PTAL

**Comment by WanLinghao** [comment]
@alejandrox1 This patch is ready, PTAL

**Review by timothysc — APPROVED** [review]
/approve 
/assign @@alejandrox1 
for final lgtm 
Please sort through the code shuffle, that often can take a while.

**Pull request merged**